spec/services/pii/re_encryptor_spec.rb

```diff
@@ -2,32 +2,45 @@
 
 RSpec.describe Pii::ReEncryptor do
   describe '#perform' do
-    it 're-encrypts PII using new code' do
+    it 're-encrypts PII onto the active profile using new code' do
       profile = create(:profile, :active, :verified, pii: { ssn: '1234' })
+
       user = profile.user
-      user_session = {
-        decrypted_pii: { ssn: '1234' }.to_json,
-      }
 
-      pii_attributes = Pii::Attributes.new
-      pii_attributes[:ssn] = '1234'
+      # make sure it doesn't use this
```

Comment:
Could we throw a better explanation with some context onto this, e.g.

> If the user has an active profile and a pending profile then the ReEncryptor is expected to only re-encrypt the active profile so the personal key for the active profile can be displayed to the user. To ensure this this test creates a pending profile to ensure that it's PII is not what is recovered when we call `Profile#recover_pii`

Reply:
Done.